tsconfig.json

```diff
@@ -4,7 +4,7 @@
   "compilerOptions": {
     "moduleResolution": "node",
     "target": "es6",
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
You're putting `exnext` here but then the `tsc` command gets called with `tsc --module commonjs`

Reply:
@wmhilton @XVincentX 
This config is used by almost all (if not all) projects using `@stoplight/scripts`, so it's usage is actually broader than it may seem to.
The reason I set `esnext` here it to make sure that's a default for all projects. Unfortunately, oclif doesn't have es modules.

Reply:
By the way I do not know about your experience but I personally thing that oclif, to be honest, is not *that* cool after all.

Reply:
I don't like oclif as well. Always used `yargs` and it's by far better.
Although it may not provide such extensive functionality as oclif does, it's just less opinionated.
I don't like the fact oclif provides its own binary script - while it lets you setup CLI quicker without any hassle, it tends to be annoying in a longer run.

Reply:
To me the "meh" moment was when I discovered that, although they claim to be supporting Typescript, they _do not even load your tsconfig file_

And I do not like their class based approach with static methods. Anyway, rant over, I think this is good to go, right?

Reply:
I'll adjust rollup config a bit - will create a rollup.config.js in a template and 'reverse' the way we merge conflict